src/OrchardCore.Modules/OrchardCore.Workflows/Views/WorkflowTrimming.Fields.Edit.cshtml

```diff
@@ -0,0 +1,113 @@
+@using OrchardCore.Workflows.Models
+@using OrchardCore.Workflows.Trimming.Services
+@using OrchardCore.Workflows.Trimming.ViewModels
+@model WorkflowTrimmingViewModel
+
+<section>
```

Comment:
We usually done use section. Do you really need this?

Reply:
I don't think it's needed. Note that I didn't add this.

Reply:
thanks for removing it.